packages/psss/src/containers/Tables/CountryTable.js

```diff
@@ -140,29 +145,49 @@ const countryColumns = [
   },
 ];
 
-export const CountryTable = React.memo(({ data, isLoading, errorMessage, page, setPage }) => (
-  <ExpandableTable
-    isLoading={isLoading}
-    columns={countryColumns}
-    data={data}
-    errorMessage={errorMessage}
-    onChangePage={setPage}
-    page={page}
-    Body={CountryTableBody}
```

Comment:
This custom CountryTable body was built to handle the expanded state of the table. It turns out the only reason it was needed was because there was a bug causing the table to be re-mounted constantly. Fixing the bug means the table now takes care of its open and closed state itsself.

Reply:
Great stuff!